pkg/vsphere/rbac/rbac.go

```diff
@@ -393,6 +394,20 @@ func (am *AuthzManager) getRoleName(resource *Resource) string {
 	}
 }
 
+func isNotSupportedError(ctx context.Context, err error) bool {
```

Comment:
Feels like this and isNotFound should roll into base govmomi somehow. @dougm?
Not needed for this PR unless already available in govmomi.

Reply:
govmomi probably should have some of these error helpers, but it doesn't other than `types.IsFileNotFound` that VIC already makes use of.